test/e2e/events_test.go

```diff
@@ -222,6 +222,38 @@ var _ = Describe("Podman events", func() {
 		Expect(result2.OutputToString()).To(ContainSubstring(fmt.Sprintf("pod_id=%s", id)))
 	})
 
+	It("podman events network connection", func() {
+		SkipIfRootless("Network creation not supported in rootless")
```

Comment:
I don't think this is actually true?

Reply:
```
Error: "slirp4netns" is not supported: invalid network mode
```

Reply:
Yeah this should not be skipped, you must create the container with `--network bridge` to support connect/disconnect